charge.js.org

brandonweiss opened the pull request:
- [x] There is reasonable content on the page (see: [No Content](https://github.com/js-org/js.org/wiki/No-Content))
- [x] I have read and accepted the [Terms and Conditions](http://js.org/terms.html)

I’d like to use this subdomain for the docs for [Charge](https://github.com/brandonweiss/charge). Thanks!

P.S. There is content but it looks broken and unstyled because GitHub uses a sub-directory (brandoweiss.github.io/charge) but the stylesheet and image links use absolute URLs (without a domain/hostname). As soon as it’s using a custom sub-domain all the links will work correctly!

brandonweiss commented:
That’s weird, I do have one… I’m not sure what happened. I just checked and it’s there.

https://github.com/brandonweiss/charge/blob/gh-pages/CNAME

MattIPv4 commented:
@brandonweiss https://brandonweiss.github.io/charge/ isn't redirecting to the js.org domain in the CNAME file as expected. Please check your GitHub pages settings to ensure it has detected the custom domain correctly.

brandonweiss commented:
@MattIPv4 Oh, sorry! I wasn’t exactly sure what order to do that in. I wasn’t sure how you’d verify the presence of content if I added the custom domain first.

I just added the custom domain in the project settings!

indus commented:
Welcome https://charge.js.org

brandonweiss commented:
@indus Thanks! ❤️
